modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -566,199 +542,6 @@ function inope_policy_review_update_query(&$form_state) {
 }
 
 /**
- * Retrieve admin table from cache if possible.
```

Comment:
One thing I would hoped you would have brought up in the client conversation yesterday was that caching was going to be lost -- or that it would change - depending on what your plans are for that.

Then again looking at this -- the caching with the sorts was something we talked about redoing in our last demo - so it's probably ok to say goodbye to this particular implementation.